tests/06-lb.sh

```diff
@@ -300,9 +221,9 @@ CLANG_OPTS="-D__NR_CPUS__=$(nproc) -DLB_L3 -DLB_REDIRECT=$NH_IFINDEX -DLB_DSTMAC
 touch netdev_config.h
 clang $CLANG_OPTS -c $LIB/bpf_lb.c -o tmp_lb.o
```

Comment:
We already have a script that searches for llvm. I will hook it into the test scripts.